interface/main/holidays/Holidays_Storage.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+/**
+ * User: sharonco@matrix.co.il
+ * Date: 6/15/16
+ * This class contains all the interaction with the database that are used by the holidays/clinic closed events
+ */
+class Holidays_Storage{
+    const TABLE_NAME = "calendar_external";
+    const CALENDAR_CATEGORY_HOLIDAY = "6";
+    const CALENDAR_CATEGORY_CLOSED = "7";
+
+    /**
+     * This function selects ALL the holidays from the table calendar_external and returns an array
+     * @return array
+     */
+    public function get_holidays(){
+        $holidays= array();
+
+        $sql = sprintf('SELECT * FROM %s', self::TABLE_NAME);
```

Comment:
If there is potential for the self::TABLE_NAME to be modified, then would need to prevent sql-injection via this function:
https://github.com/openemr/openemr/blob/master/library/formdata.inc.php#L138